zero/gk_geometry.c

```diff
@@ -78,11 +78,11 @@ struct gkyl_rect_grid gkyl_gk_geometry_augment_grid(struct gkyl_rect_grid grid,
     cells[1] = 1;
     cells[2] = grid.cells[0];
 
-    lower[0] = geometry.world[0] - 1e-5;
+    lower[0] = geometry.world[0] - fmin(1e-5, geometry.world[0]*0.1);
     lower[1] = geometry.world[1] - 1e-1;
     lower[2] = grid.lower[0];
 
-    upper[0] = geometry.world[0] + 1e-5;
+    upper[0] = geometry.world[0] + fmin(1e-5, geometry.world[0]*0.1);
```

Comment:
This is good. The finite difference uses 1e-6*grid.dx[0] in the psi direction. This will only get used  when the flux surface is at a very small value (< 1e-5) so I think 10% is safe and good.